From: Simon McVittie <smcv@debian.org>
Date: Mon, 6 Sep 2021 10:40:23 +0100
Subject: test: Fix path used in AC_CONFIG_AUX_DIR

AC_CONFIG_AUX_DIRS is an undocumented, internal version of
AC_CONFIG_AUX_DIR that takes a whitespace-separated list, instead of a
single path to add to the list. Use the documented version instead, as
in the top-level configure.ac.

Newer versions of autoconf treat the argument to AC_CONFIG_AUX_DIRS
as being literal (they do not expand the shell variable), causing
autoreconf to fail. The argument to AC_CONFIG_AUX_DIR is documented
to be relative to $srcdir anyway, so there is no need to specify $srcdir
a second time.

Bug: https://github.com/libsdl-org/SDL/issues/4719
Signed-off-by: Simon McVittie <smcv@debian.org>
Forwarded: https://github.com/libsdl-org/SDL/pull/4720
---
 test/configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/configure.ac b/test/configure.ac
index e2067e3..a3940de 100644
--- a/test/configure.ac
+++ b/test/configure.ac
@@ -3,7 +3,7 @@ AC_INIT
 AC_CONFIG_SRCDIR([loopwave.c])
 
 dnl Detect the canonical build and host environments
-AC_CONFIG_AUX_DIRS($srcdir/../build-scripts)
+AC_CONFIG_AUX_DIRS([../build-scripts])
 AC_CANONICAL_HOST
 
 dnl Check for tools
